src/Jackalope/Transport/DoctrineDBAL/CachedClient.php

```diff
@@ -147,10 +205,14 @@ public function getAccessibleWorkspaceNames()
      */
     public function getNode($path)
     {
+        if (empty($this->caches['nodes'])) {
+            return parent::getNode($path);
```

Comment:
correct .. but the Client should also do any real work for this either .. so I figured its faster to do it like this, than to even bother to look at the cache backend.